frontend/src/app/reducers/browser.js

```diff
@@ -1,9 +1,12 @@
 import { handleActions } from 'redux-actions';
+import config from '../config';
+import { isMinFirefoxVersion } from '../lib/utils';
 
-const setIsFirefox = (state, { payload: isFirefox }) => ({ ...state, isFirefox });
+const userAgent = navigator.userAgent.toLowerCase();
+const isUserAgentFirefox = userAgent.indexOf('firefox') > -1;
```

Comment:
This is already merged, but I think this is something we'll need to fix: This stuff should probably have been in `index.js` where the `setFirefox` action was. Add a `setFirefoxVersion` action and do that from there too